addons/hr/static/tests/web/m2x_avatar_user_tests.js

```diff
@@ -63,22 +63,18 @@ QUnit.module("M2XAvatarUser", ({ beforeEach }) => {
             job_title: "sub manager",
             department_id: departmentId,
         });
-        const mockRPC = (route, args) => {
-            if(route === "/web/dataset/call_kw/res.users/read"){
-                assert.deepEqual(args.args[1], [
-                "name", 
-                "email", 
-                "phone", 
-                "im_status",
-                "share",
-                "work_phone",
-                "work_email",
-                "job_title",
-                "department_id",
-                "employee_parent_id",
-                "employee_ids"]);
-                assert.step("user read");
+
+        const mockRPC = async (route, args, originalRPC) => {
+            if(route === "/mail/avatar_card/get_user_info") {
+                const user_info = await originalRPC(route, args);
+                const user = pyEnv["res.users"].searchRead([["id", '=', args["user_id"]]])[0];
+                user_info.job_title = user.job_title;
+                user_info.department_id = user.department_id;
+                user_info.work_phone = user.work_phone;
+                user_info.work_email = user.work_email;
+                return user_info;
             }
+            return await originalRPC(route, args);
```

Comment:
Couldn't this be in a patch of `/mail/avatar_card/get_user_info` on the mock server?

If no test poses much issue, this mock by hand on this test looks like what should be by default for any unit test relying on `hr` module.